Code/JavaWrappers/RDKFuncs_doc.i

```diff
@@ -89,7 +89,7 @@ If the conformer provided is not a 3D conformer, nothing will be done.
 */
 public";
 
-%javamethodmodifiers RDKit::MolOps::assignStereochemistry 	( 	ROMol &  	mol, 		bool  	cleanIt = false, 		bool  	force = false	  	) 			"
+%javamethodmodifiers RDKit::MolOps::assignStereochemistry 	( 	ROMol &  	mol, 		bool  	cleanIt = false, 		bool  	force = false, 		UINT_VECT *  	stereocenters = 0, 		bool  	onlyStereoAny = false	  	) 			"
```

Comment:
It's better (though not strictly necessary) if the default value of `stereocenters` is `NULL` instead of `0`